Jenkinsfile

```diff
@@ -43,6 +43,10 @@ node {
             sh "${gradleCommand} :apix-interface:build :apix-interface:javadoc"
         }
 
+        stage("Unit test apix-rest-v1") {
+            sh "${gradleCommand} :apix-rest-v1:test"
```

Comment:
(non-blocking) Did you check whether unit tests already ran implicitly (as gradle task dependency)? I would kind of expect it to...

Reply:
The gradle output doesnt list it as executed task, and there are no entries in the test results on master. So no, I dont think tests are implicitly linked to regular build tasks.

Reply:
Since these are all sequential stages, I don't understand why all the tests are split up.

Ideally, tests are run with a *single* command `./gradlew check`, which runs the `check` task in all projects. This meta-task should be configured to depend on all the test tasks (it depends on `test` automatically, but other tasks like `integrationTest` should be wired manually as well)

Reply:
@vierbergenlars I think (this was before my time) one reason is that most tests are not unit tests, and thus not run by `gradle check`.

Reply:
One thing I do want to look into if we move this build to GH Actions, is to parallellize every version. May be better to keep integration tests run as part of every version build.